From 7823291ac45cd1f7fb7d975d4529cca269faf5de Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Krzysztof=20Wilczy=C5=84ski?= <kwilczynski@kernel.org>
Date: Tue, 21 Jul 2026 02:04:24 +0000
Subject: [PATCH 10643/15333] PCI/sysfs: Add pci_ prefix to static PCI resource
 attribute names
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Currently, the static binary attributes for the PCI resource files are
generated with the names dev_resource<N>_io_attr, dev_resource<N>_uc_attr
and dev_resource<N>_wc_attr.

The macros that generate these attributes and the arrays that collect them
already carry the pci_ prefix, as do the sibling legacy I/O and memory
attributes, such as pci_legacy_io_attr.  Only the generated variable names
lack it.

Rename the generated variables to pci_dev_resource<N>_io_attr,
pci_dev_resource<N>_uc_attr and pci_dev_resource<N>_wc_attr, and update the
attribute pointer arrays to match.

While at it, re-align the continuation backslashes in the resource
attribute macros to match.

No functional changes intended.

Signed-off-by: Krzysztof Wilczyński <kwilczynski@kernel.org>
[bhelgaas: shorten pci_dev_resource##_bar##_wc_attr to fit in 80 columns]
Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
Link: https://patch.msgid.link/20260721020427.1541197-2-kwilczynski@kernel.org
---
 drivers/pci/pci-sysfs.c | 50 ++++++++++++++++++++---------------------
 1 file changed, 25 insertions(+), 25 deletions(-)

diff --git a/drivers/pci/pci-sysfs.c b/drivers/pci/pci-sysfs.c
index 6b016792e0ad..91f8f15beaa5 100644
--- a/drivers/pci/pci-sysfs.c
+++ b/drivers/pci/pci-sysfs.c
@@ -1284,9 +1284,9 @@ static loff_t pci_llseek_resource(struct file *filep,
  * attribute, it's not going to work, so override it as well.
  */
 #if arch_can_pci_mmap_io()
-# define __PCI_RESOURCE_IO_MMAP_ATTRS		\
-	.f_mapping = iomem_get_mapping,		\
-	.llseek = pci_llseek_resource,		\
+# define __PCI_RESOURCE_IO_MMAP_ATTRS	\
+	.f_mapping = iomem_get_mapping,	\
+	.llseek = pci_llseek_resource,	\
 	.mmap = pci_mmap_resource_uc,
 #else
 static int pci_mmap_resource_io_unsupported(struct file *filp,
@@ -1302,7 +1302,7 @@ static int pci_mmap_resource_io_unsupported(struct file *filp,
 #endif
 
 #define pci_dev_resource_io_attr(_bar)					\
-static const struct bin_attribute dev_resource##_bar##_io_attr = {	\
+static const struct bin_attribute pci_dev_resource##_bar##_io_attr = {	\
 	.attr = { .name = "resource" __stringify(_bar), .mode = 0600 },	\
 	.private = (void *)(unsigned long)(_bar),			\
 	.read = pci_read_resource,					\
@@ -1311,7 +1311,7 @@ static const struct bin_attribute dev_resource##_bar##_io_attr = {	\
 }
 
 #define pci_dev_resource_uc_attr(_bar)					\
-static const struct bin_attribute dev_resource##_bar##_uc_attr = {	\
+static const struct bin_attribute pci_dev_resource##_bar##_uc_attr = {	\
 	.attr = { .name = "resource" __stringify(_bar), .mode = 0600 },	\
 	.private = (void *)(unsigned long)(_bar),			\
 	.f_mapping = iomem_get_mapping,					\
@@ -1319,8 +1319,8 @@ static const struct bin_attribute dev_resource##_bar##_uc_attr = {	\
 	.mmap = pci_mmap_resource_uc,					\
 }
 
-#define pci_dev_resource_wc_attr(_bar)					      \
-static const struct bin_attribute dev_resource##_bar##_wc_attr = {	      \
+#define pci_dev_resource_wc_attr(_bar)						\
+static const struct bin_attribute pci_dev_resource##_bar##_wc_attr = {	      \
 	.attr = { .name = "resource" __stringify(_bar) "_wc", .mode = 0600 }, \
 	.private = (void *)(unsigned long)(_bar),			      \
 	.f_mapping = iomem_get_mapping,					      \
@@ -1406,32 +1406,32 @@ pci_dev_resource_wc_attr(4);
 pci_dev_resource_wc_attr(5);
 
 static const struct bin_attribute *const pci_dev_resource_io_attrs[] = {
-	&dev_resource0_io_attr,
-	&dev_resource1_io_attr,
-	&dev_resource2_io_attr,
-	&dev_resource3_io_attr,
-	&dev_resource4_io_attr,
-	&dev_resource5_io_attr,
+	&pci_dev_resource0_io_attr,
+	&pci_dev_resource1_io_attr,
+	&pci_dev_resource2_io_attr,
+	&pci_dev_resource3_io_attr,
+	&pci_dev_resource4_io_attr,
+	&pci_dev_resource5_io_attr,
 	NULL,
 };
 
 static const struct bin_attribute *const pci_dev_resource_uc_attrs[] = {
-	&dev_resource0_uc_attr,
-	&dev_resource1_uc_attr,
-	&dev_resource2_uc_attr,
-	&dev_resource3_uc_attr,
-	&dev_resource4_uc_attr,
-	&dev_resource5_uc_attr,
+	&pci_dev_resource0_uc_attr,
+	&pci_dev_resource1_uc_attr,
+	&pci_dev_resource2_uc_attr,
+	&pci_dev_resource3_uc_attr,
+	&pci_dev_resource4_uc_attr,
+	&pci_dev_resource5_uc_attr,
 	NULL,
 };
 
 static const struct bin_attribute *const pci_dev_resource_wc_attrs[] = {
-	&dev_resource0_wc_attr,
-	&dev_resource1_wc_attr,
-	&dev_resource2_wc_attr,
-	&dev_resource3_wc_attr,
-	&dev_resource4_wc_attr,
-	&dev_resource5_wc_attr,
+	&pci_dev_resource0_wc_attr,
+	&pci_dev_resource1_wc_attr,
+	&pci_dev_resource2_wc_attr,
+	&pci_dev_resource3_wc_attr,
+	&pci_dev_resource4_wc_attr,
+	&pci_dev_resource5_wc_attr,
 	NULL,
 };
 
-- 
2.55.0

